update.ps1

```diff
@@ -1,43 +1,34 @@
 #####################################################
 # HelloID-Conn-Prov-Target-SDBHR-Update
-#
+# PowerShell V2
 # Version: 1.0.0
+# See https://api.sdbstart.nl/swagger/ui/index#!/Medewerkers/Medewerkers_Put for supported properties
 #####################################################
-# Initialize default values
-$c = $configuration | ConvertFrom-Json
-$p = $person | ConvertFrom-Json
-$aRef = $accountReference | ConvertFrom-Json
-$success = $false # Set to false at start, at the end, only when no error occurs it is set to true
-$auditLogs = [System.Collections.Generic.List[PSCustomObject]]::new()
+#$actionContext.DryRun = $false
 
-# Set TLS to accept TLS, TLS 1.1 and TLS 1.2
-[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls -bor [Net.SecurityProtocolType]::Tls11 -bor [Net.SecurityProtocolType]::Tls12
+#$MutationDate = $actionContext.Data.beginDatum.Substring(0,10) # startdate from contract
```

Comment:
I assume these are settings used for testing. If that’s the case, please clean this up. If not, adjust the code to correctly calculate mutationdate.

Reply:
fixed